api-spec/definitions/STAC-collections.fragment.yaml

```diff
@@ -28,7 +28,7 @@ components:
         version:
           title: Collection Version
           type: string
-          example: 1
```

Comment:
I was told by a dataset creator that the Major.Minor.Patch is very software-centered and datssets are usually not versions like that. So the single 1 instead 1.0.0 was on purpose.

Reply:
Ah, makes sense.

Are they versioned by date? I was thinking of using an example that's obviously a string (even if it's declared so immediately above).

Reply:
As far as I know it depends on the provider, but an incrementing ID or date seems more reasonable than the Major.Minor.Patch.